_data/en/settings.yml

```diff
@@ -368,13 +366,6 @@ policies:
         - text: Privacy Impact Assessment
           url: '#impact'
       url: '/policy/our-privacy-act-statement/'
-    - name: Our security practices
-      sublinks:
-        - text: Vulnerability disclosure policy
-          url: '#vdp'
-        - text: For more information
-          url: '#more-info'
-      url: '/policy/our-security-practices/'
```

Comment:
I'll follow-up separately, but I don't think this data is actually used for anything, and we could remove the entire top-level `policies` section from these data files.